hoomd/md/pytest/test_methods.py

```diff
@@ -273,6 +285,92 @@ def test_npt_attributes_attached_3d(simulation_factory,
     npt.gamma = 2.0
     assert npt.gamma == 2.0
 
+    assert npt.translational_thermostat_dof == (0.0, 0.0)
+    npt.translational_thermostat_dof = (0.125, 0.5)
+    assert npt.translational_thermostat_dof == (0.125, 0.5)
+
+    assert npt.rotational_thermostat_dof == (0.0, 0.0)
+    npt.rotational_thermostat_dof = (0.5, 0.25)
+    assert npt.rotational_thermostat_dof == (0.5, 0.25)
+
+    assert npt.barostat_dof == (0.0, 0.0, 0.0, 0.0, 0.0, 0.0)
+    npt.barostat_dof = (1.0, 2.0, 4.0, 6.0, 8.0, 10.0)
+    assert npt.barostat_dof == (1.0, 2.0, 4.0, 6.0, 8.0, 10.0)
+
+
+def test_npt_thermalize_extra_dof(simulation_factory,
+                                  two_particle_snapshot_factory):
+    """Tests that NPT.thermalize_extra_dof can be called."""
+    all_ = hoomd.filter.All()
+    constant_t = hoomd.variant.Constant(2.0)
+    constant_s = [1, 2, 3, 0.125, 0.25, 0.5]
+    npt = hoomd.md.methods.NPT(filter=all_,
+                               kT=constant_t,
+                               tau=2.0,
+                               S=constant_s,
+                               tauS=2.0,
+                               box_dof=[True, True, True, True, True, True],
+                               couple='xyz')
+
+    sim = simulation_factory(two_particle_snapshot_factory())
+    sim.operations.integrator = hoomd.md.Integrator(0.005, methods=[npt])
+    sim.operations.schedule()
+
+    npt.thermalize_extra_dof(100)
+    xi, eta = npt.translational_thermostat_dof
+    assert xi != 0.0
+    assert eta == 0.0
+
+    xi_rot, eta_rot = npt.rotational_thermostat_dof
+    assert xi_rot == 0.0
+    assert eta_rot == 0.0
```

Comment:
You can always validate the tuple directly (since IIRC it is a tuple and not a list), but no need to change it now, more for future reference.

Reply:
Right, I made it explicit when I was being careful to ensure that the two variables appeared in the correct order in the tuple. They have different layouts in C++ between NVT and NPT.